packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -63,12 +63,10 @@ export class TerminalFrontendContribution implements CommandContribution, MenuCo
     }
 
     protected async newTerminal(): Promise<void> {
-        const newTerminal = await this.widgetManager.getOrCreateWidget(TERMINAL_WIDGET_FACTORY_ID, <TerminalWidgetFactoryOptions>{
+        await this.widgetManager.getOrCreateWidget(TERMINAL_WIDGET_FACTORY_ID, <TerminalWidgetFactoryOptions>{
             created: new Date().toString()
         });
-
-        this.app.shell.addToMainArea(newTerminal);
-        this.app.shell.activateMain(newTerminal.id);
+        return Promise.resolve();
```

Comment:
you are in async already, so no need for `Promise.resolve()`.

Reply:
Right got to watch-out for that thx